pkg/upgrade/upgrade.go

```diff
@@ -275,3 +276,30 @@ func upgradeToVer3(ctx context.Context, cli *clientv3.Client) error {
 func upgradeToVer4(cli *clientv3.Client, uctx Context) error {
 	return nil
 }
+
+// upgradeToVer5 does upgrade operations from Ver4 to Ver5 (v2.0.7) to update online-ddl from online-ddl-scheme in configure.
```

Comment:
```suggestion
// upgradeToVer5 does upgrade operations from Ver4 to Ver5 (v2.1.0) to update online-ddl from online-ddl-scheme in configure.
```

OK to not change, we may change the release version in future 😂